src/textual/_terminal_features.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import annotations
+from typing import NamedTuple
+
+from textual._ansi_sequences import TERMINAL_MODES_ANSI_SEQUENCES
+from textual._terminal_modes import Mode
+
+
+class TerminalSupportedFeatures(NamedTuple):
+    """
+    Handles information about the features the current terminal emulator seems to support.
+    """
+
+    mode2026_synchronized_update: bool = False
+    """@link https://gist.github.com/christianparpart/d8a62cc1ab659194337d73e399004036"""
+
+    @classmethod
+    def from_autodetect(cls) -> TerminalSupportedFeatures:
```

Comment:
This doesn't seem to do anything (currently). And I don't think we would be able to autodetect much outside of the driver.

Perhaps this should be a simple structure to record the supported features, without much functionality.

Reply:
Indeed, that's a remnant of the stage where this class was able to manage both the "mode 2026" and the iTerm2 own sync mode - in which case it was able to set the mode for iTerm2 by using env vars.
I can totally remove that method now that its implementation is empty :slightly_smiling_face: